### PR TITLE
Vetro: update alignment and use columns to collapse better on mobile

### PR DESCRIPTION
**Pull request by jffng**
#### Changes proposed in this Pull Request:

This PR updates fixes a number of alignment issues with Vetro.

- Fix an issue with the "about" text alignment on the index template
- Uses columns instead of a group to achieve the same padding on mobile for page and single templates
- Fixes 404, archive, and search templates because content was previously hitting edges of the screen 

Note I discovered a bug in the site editor with columns + site wide padding that I'm still investigating: 

Editor (does not have padding correctly applied) | View (correct padding)
--- | ---
Screenshot 2022-12-22 at 10 45 56 AM | Screenshot 2022-12-22 at 10 46 00 AM

#### Related issue(s):

See the feedback in https://github.com/Automattic/themes/pull/6725#issuecomment-1347963966

cc @henriqueiamarino

### CODE REVIEW

File: vetro/patterns/hidden-404.php [hunk 1]
```diff
@@ -6,18 +6,18 @@
...
-<div class="wp-block-group" style="margin-top:0px;margin-bottom:0px;padding-top:160px;padding-right:0px;padding-bottom:0px;padding-left:0px"><!-- wp:group {"align":"wide","style":{"spacing":{"blockGap":"0px","padding":{"top":"0px","right":"0px","bottom":"0px","left":"0px"},"margin":{"top":"0px","bottom":"0px"}}}} -->
-<div class="wp-block-group alignwide" style="margin-top:0px;margin-bottom:0px;padding-top:0px;padding-right:0px;padding-bottom:0px;padding-left:0px"><!-- wp:heading {"textAlign":"left","align":"wide","style":{"typography":{"fontWeight":"600","lineHeight":"1","fontStyle":"normal","fontSize":"8rem"}},"className":"has-text-align-center"} -->
+<!-- wp:group {"tagName":"main","style":{"spacing":{"padding":{"top":"160px","bottom":"0px"},"blockGap":"0px","margin":{"top":"0px","bottom":"0px"}}},"layout":{"type":"constrained"}} -->
+<main class="wp-block-group" style="margin-top:0px;margin-bottom:0px;padding-top:160px;padding-bottom:0px"><!-- wp:group {"align":"wide","style":{"spacing":{"blockGap":"0px","padding":{"top":"0px","bottom":"0px"},"margin":{"top":"0px","bottom":"0px"}}}} -->
```
Comment (mikachan) [inline vetro/patterns/hidden-404.php:10]: This div has been changed to a main, but I think an ending main tag still needs to be added. I think it's probably the last /div on line 30 that needs to be changed.

**Review by mikachan — COMMENTED** [review]
This is looking good to me, I can see the alignments you mentioned look better. I've just left one minor comment.

File: vetro/patterns/hidden-404.php (hunk 1 above)
Comment (jffng) [reply vetro/patterns/hidden-404.php:10]: Oh good catch, thanks! Updated in [cc3d901](https://github.com/Automattic/themes/pull/6800/commits/cc3d9010fcf04da1383db4bff47fc62e8b43eba0)

**Review by mikachan — APPROVED** [review]
Looks good!

**Pull request merged**